src/loader.c

```diff
@@ -299,6 +309,7 @@ bool parse_database(void *base, const char *name, const char *connstr)
 	db->res_pool_size = res_pool_size;
 	db->pool_mode = pool_mode;
 	db->max_db_connections = max_db_connections;
+	db->host_strategy = host_strategy;
```

Comment:
if the `host_strategy` of a database can change, should we tag it as [changed](https://github.com/pgbouncer/pgbouncer/blob/b35c9a67345e2b7789724f8974f16be565827992/src/loader.c#L300)?

Reply:
That's a good call. Yes, I think we should tag it so behavior changes after a reload. I will make that change. Thanks for the feedback!

Reply:
`host_strategy` is now reloadable.

`show databases`/`show pools` also now include `host_strategy` in their output, which makes this testable and is useful on its own.